pyOCD/tools/gdb_server.py

```diff
@@ -185,7 +187,7 @@ def list_boards(self):
 
                 # Reopen the link so we can access the USB vendor and product names from the inteface.
                 # If it's not a USB based link, then we don't attempt this.
-                if isinstance(mbed.link, DAPAccessUSB):
+                if isinstance(mbed.link, DAPAccessCMSISDAP):
```

Comment:
Yes I think that ws-interface will have product_name and vendor_name set to empty string. Unless we can get this info from the board.